components/IndividualPothole.js

```diff
@@ -28,18 +25,19 @@ class IndividualPothole extends React.Component {
     super();
     this.state = {
       upvotes: 0,
-      disableUpvote: false
+      disableUpvote: false,
     };
   }
 
   async componentDidMount() {
     await this.props.getSinglePothole(this._getId());
     await this.props.getAllComments(this.props.singlePothole.id);
     // // //set # of upvoters on state
-
     this.setState({
       upvotes: this.props.singlePothole.upvoters.length,
-      disableUpvote: !!(this.props.upvoters.filter(upvoter => upvoter.id === this.props.userId).length)
+      disableUpvote: !!this.props.upvoters.filter(
```

Comment:
Try find() instead. It will break out as soon as it is true or return undefined, which will be coerced to false with !!

Reply:
I like it! Changed